iroh-gossip/Cargo.toml

```diff
@@ -34,10 +34,11 @@ once_cell = "1.18.0"
 genawaiter = { version = "0.99.1", default-features = false, features = ["futures03"] }
 
 [dev-dependencies]
-tracing-subscriber = { version = "0.3", features = ["env-filter"] }
 clap = { version = "4", features = ["derive"] }
-url = "2.4.0"
+iroh-test = { path = "../iroh-test" }
 rand_chacha = "0.3.1"
+tracing-subscriber = { version = "0.3", features = ["env-filter"] }
```

Comment:
is this direct dependency still needed?

Reply:
Yes, because of examples.  But good catch, some others are no longer needed and I cleaned them up.